src/common/config_parser.cpp

```diff
@@ -512,6 +512,9 @@ void ConfigParser::addOptionsTranslation(cli::CLIWrapper& cli) {
   cli.add<std::string>("--trie-pruning-path",
      "Use trie pruning during translation. Provide path to the monolingual corpora.")
     ->implicit_val("");
+  cli.add<std::string>("--paraphrase",
```

Comment:
We should add a check to make sure that there is an error if `paraphrase` is enabled. This could be done around here: https://github.com/marian-nmt/marian-dev/blob/trieme_CPUonly/src/translator/translator.h#L52

But I can add it later. It's not critical since this code ain't production anyways.